client/src/modules/cohort/model/actions.js

```diff
@@ -107,32 +107,32 @@ const removeMemberFailure = () => ({
   type: CohortActionTypes.REMOVE_MEMBER_FAILURE
 });
 
-const removeMemberSuccess = (cohortId, userId) => ({
+const removeMemberSuccess = payload => ({
```

Comment:
IMHO, you loose "typings" here - the expected parameters are now only defined in the call places. I would revert that.